src/index.js

```diff
@@ -1,10 +1,12 @@
+import { flatten, zip } from './util';
+import updateBlockquote from './blockquoteTokens';
+
 export default function markdownItAttribution(md, options) {
   const attributionPrefix = (options && options.attributionPrefix) || '--';
   let Token;
 
-  function setupBlockquoteRule() {
-    md.core.ruler.after('block', 'attribution', blockquoteRule);
-  }
+  // setup Blockquote Rule
+  md.core.ruler.after('block', 'attribution', blockquoteRule);
```

Comment:
shouldn't we define the functions up top? 
Why do we define `blockquoteRule` afterwards?